java/vector/src/main/java/org/apache/arrow/vector/complex/AbstractContainerVector.java

```diff
@@ -112,4 +112,12 @@ public ListVector addOrGetList(String name) {
   public UnionVector addOrGetUnion(String name) {
     return addOrGet(name, FieldType.nullable(MinorType.UNION.getType()), UnionVector.class);
   }
+
+  @Override
+  public void copyFrom(int fromIndex, int thisIndex, ValueVector from) {
```

Comment:
exception if not supported ?

Reply:
Revised. Thanks for the good suggestion.